open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/properties/AdminServicesConfigHeader.java

```diff
@@ -35,9 +35,7 @@
         @JsonSubTypes.Type(value = OpenLineageServerConfig.class, name = "OpenLineageConfig"),
         @JsonSubTypes.Type(value = RepositoryServicesConfig.class, name = "RepositoryServicesConfig"),
         @JsonSubTypes.Type(value = ResourceEndpointConfig.class, name = "ResourceEndpointConfig"),
-        @JsonSubTypes.Type(value = SecuritySyncConfig.class, name = "SecuritySyncConfig"),
         @JsonSubTypes.Type(value = StewardshipEngineServicesConfig.class, name = "StewardshipServicesConfig"),
```

Comment:
If we are talking the hit to remove obsolete config from the configuration doc then should also remove StewardshipEngineConfig, DiscoveryEngineConfig and DataPlatformConfig

Reply:
Ok I can add these

Reply:
covered by #5318 #5319